README.md

```diff
@@ -833,6 +833,7 @@ There are 52 apps that you can install on your cluster.
 | [k3sup](https://github.com/alexellis/k3sup)                                  | Bootstrap Kubernetes with k3s over SSH < 1 min.                                                                                                                   |
 | [k9s](https://github.com/derailed/k9s)                                       | Provides a terminal UI to interact with your Kubernetes clusters.                                                                                                 |
 | [kail](https://github.com/boz/kail)                                          | Kubernetes log viewer.                                                                                                                                            |
+| [kagent](https://github.com/boz/kail)                                        | Cloud Native Agentic AI                                                                                                                                          |
```

Comment:
copy and paste error here, which is odd because this table can be generated as per the footnote.

As two apps have been added there should also be some updates to the apps section